WalletWasabi.Fluent/Views/AddWallet/AddWalletPageView.axaml

```diff
@@ -26,23 +26,26 @@
       <WrapPanel Orientation="Horizontal" Margin="0 0 0 20" HorizontalAlignment="Center">
         <WrapPanel.Styles>
           <Style Selector="c|TileButton">
-            <Setter Property="Margin" Value="15"/>
-            <Setter Property="IsEnabled" Value="{Binding OptionsEnabled}"/>
+            <Setter Property="Margin" Value="4"/>
           </Style>
         </WrapPanel.Styles>
         <c:TileButton Text="Create a new wallet"
                       Classes="action"
                       Icon="{StaticResource add_regular}"
-                      Command="{Binding CreateWalletCommand}"/>
+                      Command="{Binding CreateWalletCommand}"
+                      IsEnabled="{Binding OptionsEnabled}" />
         <c:TileButton Text="Connect to hardware wallet"
                       Icon="{StaticResource calculator_regular}"
-                      Command="{Binding ConnectHardwareWalletCommand}"/>
+                      Command="{Binding ConnectHardwareWalletCommand}"
+                      IsEnabled="{Binding OptionsEnabled}" />
         <c:TileButton Text="Import a wallet"
                       Icon="{StaticResource import_regular}"
-                      Command="{Binding ImportWalletCommand}"/>
+                      Command="{Binding ImportWalletCommand}"
+                      IsEnabled="{Binding OptionsEnabled}" />
         <c:TileButton Text="Recover a wallet"
                       Icon="{StaticResource recover_arrow_right_regular}"
-                      Command="{Binding RecoverWalletCommand}"/>
+                      Command="{Binding RecoverWalletCommand}"
+                      IsEnabled="{Binding OptionsEnabled}" />
```

Comment:
I had to move the `IsEnabled` binding outside from the style because without this the TileButtons opacity was on 1 despite the `OptionsEnabled` was false.

TileButton.axaml:
```  
  <!-- Fade in/out -->
  <Style Selector="c|TileButton[IsEnabled=False]">
    <Setter Property="Opacity" Value="0.4" />
  </Style>

  <Style Selector="c|TileButton[IsEnabled=True]">
    <Setter Property="Opacity" Value="1" />
  </Style>```

Reply:
@wieslawsoltes what do you think? Batched property updates?

Reply:
> @wieslawsoltes what do you think? Batched property updates?

Looks like it
![image](https://user-images.githubusercontent.com/2297442/116616233-bed9aa80-a93c-11eb-9632-8575cb753da8.png)